### PR TITLE
Improve forks customizability

### PR DESCRIPTION
**Pull request by Soitora**
I can't test this locally due to how Vuepress builds the file, so I need to use Netlify for an accurate representation.

### CODE REVIEW

**Comment by Soitora** [comment]
Looks like everything is working pretty much as expected, safe to be reviewed:
image

File: src/.vuepress/components/ForkButtons.vue [hunk 1]
```diff
@@ -107,7 +107,7 @@ export default {
 		width 8.5rem
 		&:focus
 			outline none
-			outline-style solid
+			outline-style none
```
Comment (arkon) [inline src/.vuepress/components/ForkButtons.vue:110]: I'm usually against this since it's bad for a11y, but that can be left for a different PR.

File: src/.vuepress/components/Neko.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<template>
```
Comment (arkon) [inline src/.vuepress/components/Neko.vue:1]: These fork components all seem pretty repetitive. Can it not be abstracted into one and you just pass in some props like the class and colours?

File: src/forks/Neko.md [hunk 3]
```diff
@@ -1,18 +1,28 @@
 ---
+layout: Neko
 title: Neko
+metaTitle: Neko
+description: MangaDex specific fork of Tachiyomi and TachiyomiJ2K. This contains features specific to MangaDex that those versions cannot offer.
```
Comment (arkon) [inline src/forks/Neko.md:5]: I'd defer to Carlos's decision here, but it's more of a fork of J2K than the base app at this point.

File: src/forks/TachiyomiAZ.md [hunk 4]
```diff
@@ -1,12 +1,22 @@
 ---
+layout: TachiyomiAZ
 title: TachiyomiAZ
+metaTitle: TachiyomiAZ
+description: This fork is based off of TachiyomiEH, it has most features and fixes from Preview, as well as features from J2k.
```
Comment (arkon) [inline src/forks/TachiyomiAZ.md:5]: Could maybe mention its focus on hentai?

File: src/forks/TachiyomiJ2K.md [hunk 5]
```diff
@@ -1,10 +1,20 @@
 ---
+layout: TachiyomiJ2K
 title: TachiyomiJ2K
+metaTitle: TachiyomiJ2K
+description: This fork adds certain enhancements that is not yet present in the base app.
```
Comment (arkon) [inline src/forks/TachiyomiJ2K.md:5]: Same with this, I feel like there's a better description to be had for this fork. Up to Jay though.

File: src/.vuepress/components/Neko.vue (hunk 2 above)
Comment (Soitora) [reply src/.vuepress/components/Neko.vue:1]: I can give it a try, it was initially for if a fork wanted to change the structure, but right now I suppose no one is

File: src/forks/Neko.md (hunk 3 above)
Comment (Soitora) [reply src/forks/Neko.md:5]: That's true

File: src/forks/TachiyomiAZ.md (hunk 4 above)
Comment (Soitora) [reply src/forks/TachiyomiAZ.md:5]: I will try rewording it

File: src/forks/TachiyomiJ2K.md (hunk 5 above)
Comment (Soitora) [reply src/forks/TachiyomiJ2K.md:5]: Yeah indeed it's not a good one, but I'm extraordinarily bad with wordings

File: src/.vuepress/components/ForkButtons.vue (hunk 1 above)
Comment (Soitora) [reply src/.vuepress/components/ForkButtons.vue:110]: Planning on redoing the code for the forkbuttons component after the Home "rewrite" branch is PRd

File: src/forks/Neko.md (hunk 3 above)
Comment (Soitora) [reply src/forks/Neko.md:5]: Carlos would like to keep the wordings so I'll do just that.
